CONTRIBUTING.md

```diff
@@ -0,0 +1,134 @@
+```
+
+**Docker (Windows)**
+
+```bash
+docker run --rm --interactive --tty --volume "%cd%":/app composer update --ignore-platform-reqs --optimize-autoloader --no-plugins --no-scripts --prefer-dist
+```
+
+4. Build the runtimes you want to test using the docker build command:
+```
+cd ./runtimes/
+docker build -t {{Image Tag Name}} ./{{Folder of your runtime}}
+```
+
+3. [Follow our contribution guide to learn how you can add support for more runtimes.](https://github.com/appwrite/php-runtimes/blob/main/docs/add-runtime.md)
+
+## Introducing New Features
+
+We would 💖 for you to contribute to Open-runtimes, but we would also like to make sure Open-runtimes is as great as possible and loyal to its vision and mission statement 🙏.
+
+For us to find the right balance, please open an issue explaining your ideas before introducing a new pull request.
+
+This will allow the Open-runtimes community to have sufficient discussion about the new feature value and how it fits in the product roadmap and vision.
+
+This is also important for the Open-runtimes lead developers to be able to give technical input and different emphasis regarding the feature design and architecture. Some bigger features might need to go through our [RFC process](https://github.com/appwrite/rfc).
+
+## Other Ways to Help
+
+Pull requests are great, but there are many other areas where you can help Open-runtimes.
+
+### Blogging & Speaking
+
+Blogging, speaking about, or creating tutorials about one of Open-runtimes’s many features is great way to contribute and help our project grow.
+
+### Presenting at Meetups
+
+Presenting at meetups and conferences about your Open-runtimes projects. Your unique challenges and successes in building things with Open-runtimes can provide great speaking material. We’d love to review your talk abstract/CFP, so get in touch with us if you’d like some help!
+
+### Sending Feedbacks & Reporting Bugs
+
+Sending feedback is a great way for us to understand your different use cases of Open-runtimes better. If you had any issues, bugs, or want to share about your experience, feel free to do so on our GitHub issues page or at our [Discord channel](https://discord.gg/fP6W2qEzfQ).
+
+### Submitting New Ideas
+
+If you think Open-runtimes could use a new feature, please open an issue on our GitHub repository, stating as much information as you can think about your new idea and its implications. We would also use this issue to gather more information, get more feedback from the community, and have a proper discussion about the new feature.
+
+### Improving Documentation
+
+Submitting documentation updates, enhancements, designs, or bug fixes. Spelling or grammar fixes will be very much appreciated.
+
+### Helping Someone
+
+Searching for Open-runtimes, GitHub or StackOverflow and helping someone else who needs help. You can also help by teaching others how to contribute to Open-runtimes's repo!
```

Comment:
```suggestion
Search for Open Runtimes on GitHub or StackOverflow and help someone else who needs help. You can also help by teaching others how to contribute to Open Runtimes's repo!
```

Reply:
I think this first line needs a second review. Not entirely sure how it should be framed.